source/algorithms.tex

```diff
@@ -1126,39 +1126,39 @@
 or
 \tcode{Input\-Iterator2},
 the template argument shall satisfy the
-requirements of an input iterator\iref{input.iterators}.
+\oldconcept{InputIterator} requirements\iref{input.iterators}.
 \item
 If an algorithm's template parameter is named
 \tcode{OutputIterator},
 \tcode{OutputIterator1},
 or
 \tcode{Output\-Iterator2},
-the template argument shall satisfy the requirements
-of an output iterator\iref{output.iterators}.
+the template argument shall satisfy the
+\oldconcept{OutputIterator} requirements\iref{output.iterators}.
 \item
 If an algorithm's template parameter is named
 \tcode{ForwardIterator},
 \tcode{ForwardIterator1},
 or
 \tcode{Forward\-Iterator2},
-the template argument shall satisfy the requirements
-of a forward iterator\iref{forward.iterators}.
+the template argument shall satisfy the
+\oldconcept{ForwardIterator} requirements\iref{forward.iterators}.
```

Comment:
The [forward.iterators] section is not so named, and the table of that name doesn't show semantics.

Reply:
Repeating my reply to your original comment (that Github will likely bury after the `\oldconcept` push):

"forward iterators" will be ambiguous once Ranges are merged. I understand your point about the distinction between the table and the additional requirements outside the table, though. Do you agree it would be clearer if I retitle the existing sections e.g. "Cpp98ForwardIterator requirements"?

Reply:
I'm ok with that, although that doesn't really remove the ambiguity if the table continues to be labeled the same as the section. @zygoloid?

Reply:
Yes, I'd thought about the ambiguity of both subclauses and tables titled "Foo requirements," although that's pre-existing, for example, [allocator.requirements] is titled "Allocator requirements" and contains Table 31 "Allocator requirements". We should retitle those tables to something generic like "Allocator valid expressions with semantics" - [structure.requirements]/4 says they are "a
table that specifies an initial set of the valid expressions and their semantics." Which is also a misnomer.

I'm waiting for Richard's nod to enact these changes. I have more work to do on the merged Ranges proposal than time before the mailing deadline to do it in, so I don't want to walk down any blind alleys.